docs/supported-libraries.md

```diff
@@ -19,116 +19,116 @@ These are the supported libraries and frameworks:
-| [Apache Dubbo](https://github.com/apache/dubbo/)                                                                                            | 2.7+                          | [opentelemetry-apache-dubbo-2.7](../instrumentation/apache-dubbo-2.7/library-autoconfigure)                                                                                                                                                                                                                                                                                             |                                                                                        |
-| [Apache HttpAsyncClient](https://hc.apache.org/index.html)                                                                                  | 4.1+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     |                                                                                        |
-| [Apache HttpClient](https://hc.apache.org/index.html)                                                                                       | 2.0+                          | [opentelemetry-apache-httpclient-4.3](../instrumentation/apache-httpclient/apache-httpclient-4.3/library)                                                                                                                                                                                                                                                                               |                                                                                        |
+| [Akka Actors](https://doc.akka.io/docs/akka/current/typed/index.html)                                                                       | 2.5+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                   |
```

Comment:
I like that. There's several other instrumentations that are in the same category. 

If an instrumentation does context propagation in addition to some other semantic convention, is it worth additionally calling out the context propagation? Put another way, is "context propagation" another tag that should be considered for all instrumentations?

Reply:
this is a little tricky, e.g. some instrumentation does context propagation so that async callbacks won't appear under the client span (but under the client's parent span). take away the spans, and they _might_ not need context propagation, but we don't have any tests that would tell us this...

if you want to give it a shot I'll review them, but I'm also ok with "context propagation" only for the obvious ones (e.g. Akka, Java Executors) to start with